### PR TITLE
Add initial Rachio docs

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Source: https://github.com/Klikini/home-assistant.github.io/issues/2

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7600

### CODE REVIEW

**Comment by fabaff** [comment]
@Klikini, perhaps your iFrame configuration would be a nice addition.

File: source/_components/switch.rachio.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+The `rachio` switch platform allows you to control your [Rachio irrigations system](http://rachio.com/).
+
+Getting your Rachio API Access Token:
+
+- Log in at https://app.rach.io/
+- Click the user button at the top right
+- Click API Access Token
+- Copy it from the dialog that opens
```
Comment (Landrash) [inline source/_components/switch.rachio.markdown]: Suggest a rewrite:
>Copy the API access token from the dialog that opens.

File: source/_components/switch.rachio.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ha_category: Switch
+ha_iot_class: "Local Polling"
+ha_release: 0.46
+---
+
+The `rachio` switch platform allows you to control your [Rachio irrigations system](http://rachio.com/).
+
+Getting your Rachio API Access Token:
+
+- Log in at https://app.rach.io/
```
Comment (Landrash) [inline source/_components/switch.rachio.markdown]: Suggest changing it to be an actual link also.
```markdown
[https://app.rach.io/](https://app.rach.io/)
```

File: source/_components/switch.rachio.markdown [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+footer: true
+logo: rachio.png
+ha_category: Switch
+ha_iot_class: "Local Polling"
+ha_release: 0.46
+---
+
+The `rachio` switch platform allows you to control your [Rachio irrigations system](http://rachio.com/).
+
+Getting your Rachio API Access Token:
```
Comment (Landrash) [inline source/_components/switch.rachio.markdown]: Make this and linkable title?

```markdown
## {% linkable_title Getting your Rachio API Access Token %}
```

File: source/_components/switch.rachio.markdown [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+- Log in at https://app.rach.io/
+- Click the user button at the top right
+- Click API Access Token
```
Comment (Landrash) [inline source/_components/switch.rachio.markdown]: Dot's at the end of each line?

File: source/_components/switch.rachio.markdown (hunk 1 above)
Comment (fabaff) [inline source/_components/switch.rachio.markdown]: Thanks

**Comment by Klikini** [comment]
@fabaff 

```yaml
panel_iframe:
  rachio:
    title: Rachio
    url: "https://app.rach.io"
    icon: mdi:water-pump
```

**Comment by fabaff** [comment]
@Klikini, thanks.

**Pull request merged**